pymesos/process.py

```diff
@@ -143,14 +147,16 @@ def abort(self):
 
     def stop(self):
         Process.abort(self)
-        self.join()
-        for addr in self.conn_pool:
-            self.conn_pool[addr].close()
-        self.conn_pool.clear()
+        if threading.current_thread() != self.accept_t:
```

Comment:
i'm afraid it can introduce deadlock here. if stopme is called both in `accept_t` and `delay_t`, we may end with two threads wating for each other to finish infinitely.

in my opition, it's convienient and trouble-free to only allow the main thread to call `stop`.